webapp-django/crashstats/crashstats/static/crashstats/css/daily.less

```diff
@@ -5,6 +5,10 @@ h1 {
     font-size: 1.90em;
     font-weight: bold;
 }
+.page-heading p.new-daily-report-link {
+    float: right;
+}
+
```

Comment:
Actually, I think I prefer my way of doing it. I.e. to float it to the right. I too value consistency and whatever we decide in this PR, the final result is that it should be the same for all **4** links. 

Reasons I like my link instead:
* Floating it to the right means it doesn't have any alignment issues when it's situated near the header. E.g. see how [badly it's aligned here with the buttons to the immediate left](https://www.dropbox.com/s/ne886x407wxcl95/Screenshot%202015-10-12%2012.52.53.png?dl=0)
* Floating it to the right "indicates" that it's not a part of the main app. E.g. it's not another button to re-display the data differently. 
* By displaying it with mixed bold and mix italics gives it texture to emphasize two different words within the link. E.g. in "View the <b>new</b> <i>Crashes per user</i> report" manages to highlight the name of the report in the link and which kind it is. 
* Making the whole text italic is confusing. Why is it italic (...wonders the user)? Normally italics is used to make sub-parts of a text stand out from the rest. I.e. to emphasize. When all is italics it doesn't serve the purpose it was built for. 

What do you think about that?